exchanges/bitmex/bitmex_wrapper.go

```diff
@@ -189,8 +189,24 @@ func (b *Bitmex) SubmitOrder(p pair.CurrencyPair, side exchange.OrderSide, order
 
 // ModifyOrder will allow of changing orderbook placement and limit to
 // market conversion
-func (b *Bitmex) ModifyOrder(orderID int64, action exchange.ModifyOrder) (int64, error) {
-	return 0, common.ErrNotYetImplemented
+func (b *Bitmex) ModifyOrder(action exchange.ModifyOrder) exchange.ModifyOrderResponse {
+	var params OrderAmendParams
+
+	params.OrderID = action.OrderID
+	params.OrderQty = int32(action.Amount)
```

Comment:
The API says the OrderQty is a double, by converting to int, we're removing important decimal places!

Reply:
Can we buy and sell percentages of contracts?

Reply:
My ignorance of Bitmex is showing! You're right, but I don't like it being converted at this point. If anything, I'd rather it throw an error if there are decimal places. Otherwise, whoever calls this with a float value will not know that its decimal is being cut off (and could also round up to a value that the caller didn't specify, or have the funds for)

Reply:
added error check in new update